Pipfile

```diff
@@ -4,17 +4,17 @@ verify_ssl = true
 name = "pypi"
 
 [packages]
-marshmallow = ">=3"
-flask-marshmallow = ">=0.14"
-passlib = ">=1"
-apispec = {extras = ["yaml"], version = ">=5"}
-apispec-webframeworks = ">=0.5"
-Flask = "<2"
-Flask-RESTful = ">=0.3"
-Flask-Migrate = ">=3"
-Flask-JWT-Extended = ">=4"
+flask-marshmallow = "==0.14"
+marshmallow-sqlalchemy = "==0.25"
+passlib = "==1.7.4"
+apispec = {extras = ["yaml"], version = "==5.0"}
+apispec-webframeworks = "==0.5.2"
+Flask = "==2.1"
```

Comment:
This may be a problem. At the time I had restricted the flask version because of the requirements of taipy gui, not sure if they updated their version.

Reply:
yes, the new version for taipy-gui is Flask>=2.0 now! But I will wait for them approval ^^